### PR TITLE
Update Typography T-shirt sizing syntax

### PR DESCRIPTION
**Pull request by GarthDB**
## Description
Change the typography t-shirt size classes from `.spectrum-Heading--XXXL` to `.spectrum-Heading--sizeXXXL`

## To-do list

- [x] If my change impacts other components, I have tested to make sure they don't break.
- [ ] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [ ] This pull request is ready to merge.

### CODE REVIEW

**Comment by GarthDB** [comment]
I need to update migration guides; I just want to check VRT.

File: components/rating/metadata/rating.yml [hunk 1]
```diff
@@ -13,46 +13,46 @@ examples:
         <input class="spectrum-Rating-input" type="range" min="0" max="5" value="0" aria-label="Rating">
 
         <span class="spectrum-Rating-icon">
-          <svg class="spectrum-Icon spectrum-UIIcon-Star spectrum-Rating-starActive" focusable="false" aria-hidden="true">
+          <svg class="spectrum-Icon spectrum-UIIcon-sizeStar spectrum-Rating-starActive" focusable="false" aria-hidden="true">
```
Comment (lazd) [inline components/rating/metadata/rating.yml:16]: Regex got over excited here

File: tools/component-builder/css/processors.js [hunk 2]
```diff
@@ -205,7 +201,7 @@ function getProcessors(keepVars = false, notNested = true, secondNotNested = tru
 
           // overwrite-support for the Typography-V3 <em> & <strong> selectors
           // sharing the same classname ".spectrum-Detail". This will be added like
-          // ".spectrum-Detail--XL em {}"
+          // ".spectrum-Detail--sizXL em {}"
```
Comment (lazd) [inline tools/component-builder/css/processors.js:204]: Missing that elusive `e`

**Review by lazd — CHANGES_REQUESTED** [review]
Couple of little errors, once those are fixed I'll check it out locally and see if it's good to go.

File: components/typography/metadata/typography.yml [hunk 3]
```diff
@@ -42,22 +42,22 @@ examples:
...
     markup: |
       <div class="spectrum-Typography">
-        <h1 class="spectrum-Heading spectrum-Heading--XL">Aliquet Mauris Eu</h1>
-        <p class="spectrum-Body spectrum-Body--XL">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec eleifend est mollis ligula lobortis, tempus ultricies sapien lacinia. Nulla ut turpis velit. Sed finibus dapibus diam et sollicitudin. Phasellus in ipsum nec ante elementum congue eget in leo. Morbi eleifend justo non rutrum venenatis. Fusce cursus et lectus eu facilisis. Ut laoreet felis in magna dignissim feugiat.</p>
+        <h1 class="spectrum-Heading spectrum-Heading--sizeXL">Aliquet Mauris Eu</h1>
+        <p class="spectrum-Body spectrum-Body--sizeXL">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec eleifend est mollis ligula lobortis, tempus ultricies sapien lacinia. Nulla ut turpis velit. Sed finibus dapibus diam et sollicitudin. Phasellus in ipsum nec ante elementum congue eget in leo. Morbi eleifend justo non rutrum venenatis. Fusce cursus et lectus eu facilisis. Ut laoreet felis in magna dignissim feugiat.</p>
       </div>
 
   - name: Migrating from deprecated Typography
```
Comment (lazd) [inline components/typography/metadata/typography.yml:56]: Can we also update this section to show the new-new typography classes, not the old-new? i.e.

```
| `~.spectrum-Heading1--display~`  | `.spectrum-Heading.spectrum-Heading--sizeXXXL`                        ```

**Review by lazd — CHANGES_REQUESTED** [review]
I'm thinking if we get the migration guide updated to point at the new-new classes, and if VRT does not fail anywhere, this is good.

**Review by lazd — APPROVED** [review]

**Pull request merged**